torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4792,6 +4792,12 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            sample_inputs_func=sample_inputs_pow,
            supports_inplace_autograd=False,
            assert_autodiffed=True),
+    OpInfo('float_power',
+           dtypes=all_types_and_complex_and(torch.half, torch.bfloat16, torch.bool),
+           backward_dtypes=all_types_and_complex_and(torch.half, torch.bfloat16, torch.bool),
+           sample_inputs_func=sample_inputs_pow,
+           supports_inplace_autograd=False,
+           assert_autodiffed=False),
```

Comment:
I didn't see autograd forrmulae for `float_power` in `tools/autograd/derivatives.yaml`. Looks like autograd isn't supported for `float_power`. (`assert_autodiffed=True` gives similar error as well), please correct me if I'm wrong.

Reply:
Yes, you are right. It passes the inplace grad test after removing `supports_inplace_autograd`. I see what you mean here, though I'm curious how is it different from `assert_autodiffed`? We can discuss this offline though.

Also, removed `assert_autodiffed=False` as it's default `False`. Thanks!